package.json

```diff
@@ -53,7 +53,7 @@
     "jira-client": "^8.0.0",
     "markdown-it": "^12.3.2",
     "matrix-appservice-bridge": "^4.0.1",
-    "matrix-bot-sdk": "^0.5.19",
+    "matrix-bot-sdk": "^0.6.0",
```

Comment:
Since https://github.com/turt2live/matrix-bot-sdk/issues/247 was fixed, this can use `^0.6.1` now, right?

Reply:
Done!